exchanges/order/orders.go

```diff
@@ -208,6 +209,9 @@ func (d *Detail) UpdateOrderFromDetail(m *Detail) {
 	if d.ID == "" {
 		d.ID = m.ID
 	}
+	if d.InternalOrderID == "" {
```

Comment:
This is failing a test: ```gocryptotrader\exchanges\order\order_test.go:870: Should not be able to update the internal order ID```

Reply:
Indeed: this PR changes how the internal order ID works, where I assume the scope of the ID to be the order manager.
- UpdateOrderFromDetail updates unset order ID's - and ignores new order ID's after the first update of the order ID.
- Order IDs are always generated: when a seeding error occurs for generating a V4 UUID, a V5 ID is deterministically derived from the exchange's order ID.

I'll be so free to adapt the test, not the code.